tests/ci/benchmark_framework/compare_results.py

```diff
@@ -5,6 +5,49 @@
 import pandas as pd
 import numpy as np
 
+# Array containing ignored fields
+ignored_fields = np.array(['TrustToken-Exp1-Batch1 generate_key',
```

Comment:
Nit: create an issue in backlog to revisit this decision?

Reply:
Backlog item created (`CryptoAlg-1253`)

Reply:
Just out of curiosity, what were these `TrustToken-Exp*` fields originally benchmarking?